app/test_split_integer.py

```diff
@@ -2,20 +2,35 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 4
+    parts = split_integer(value, number_of_parts)
+    assert sum(parts) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 6
+    number_of_parts = 3
+    parts = split_integer(value, number_of_parts)
+    assert parts == [2, 2, 2]
```

Comment:
Based on clean code recommendations, will be better to do:
```suggestion
    assert split_integer(6, 3) == [2, 2, 2]
```
![image](https://github.com/mate-academy/py-split-integer/assets/87720583/f824d4c2-c43e-4d82-a797-23b3dba9641e)